students/PeterB/Session2/fizzbuzz.py

```diff
@@ -0,0 +1,11 @@
+for fizzbuzz in range(100):
+    if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
+        print("fizzbuzz")
+        continue
```

Comment:
Instead of using continues and elifs, you can achieve the same with:

if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
        print("fizzbuzz")
if fizzbuzz % 3 == 0:
        print("fizz")
if fizzbuzz % 5 == 0:
        print("buzz")